website/docs/sdk-reference/android.md

```diff
@@ -74,7 +74,7 @@ val client = ConfigCatClient.newBuilder()
 | `cache(ConfigCache)`                    | Optional, sets a custom cache implementation for the client. [More about cache](#custom-cache). |
 | `mode(PollingMode)`                     | Optional, sets the polling mode for the client. [More about polling modes](#polling-modes). |
 | `logLevel(LogLevel)`                    | Optional, defaults to `WARNING`. Sets the internal log level. [More about logging](#logging). |
-| `flagOverrides(OverrideDataSourceBuilder, OverrideBehaviour)` | Optional, configures local feature flag & setting overrides. [More about feature flag overrides](#flag-overrides). |
+| `flagOverrides(OverrideDataSourceBuilder, OverrideBehaviour)` | Optional, sets local feature flag & setting overrides. [More about feature flag overrides](#flag-overrides). |
```

Comment:
```suggestion
| `flagOverrides(OverrideDataSourceBuilder, OverrideBehaviour)` | Optional, sets the local feature flag & setting overrides. [More about feature flag overrides](#flag-overrides). |
```

At other SDKs, you added the word "the". For consistency, I suggest adding it here too.